examples/credit_loan_decisions.py

```diff
@@ -0,0 +1,1359 @@
+# the United States are subject to _legal regulations_ prohibiting
+# discrimination on the `basis of race, gender, or other protected classes
+# <https://www.govinfo.gov/content/pkg/USCODE-2011-title15/html/USCODE-2011-title15-chap41-subchapIV.htm>`_.
+# With the increasing prevalence of automated decision-systems in the financial
+# lending space, experts have raised concerns about whether these systems could
+# exacerabate existing inequalities in financial lending.
+#
+# Although the two concepts are intertwined, algorithmic fairness is not the
+# same concept as anti-discrimination law. An AI system can comply with
+# anti-discrimination law while exhibiting fairness-related concerns. On the
+# other hand, some fairness interventions may be illegal under
+# anti-discrimination law. `Xiang and Raji  (2019)
+# <https://arxiv.org/abs/1912.00761>`_` discuss the compatibilities and
+# disconnects between anti-discrimination law and algorithmic notions of
+# fairness. In this case study, we focus on fairness in financial services
+# rather than compliance with financial anti-discrimination regulations.
+
+# %%
+# *EY* case study
+# ==================
+
+# %%
+# In this case study, we aim to replicate the work done in a `white paper
+# <https://www.microsoft.com/en-us/research/uploads/prod/2020/09/Fairlearn-EY_WhitePaper-2020-09-22.pdf>`_,
+# co-authored by _Microsoft_ and _EY_, on mitigating gender-related performance
+# disparities in financial lending decisions. In their analysis, _Microsoft_ and
+# _EY_ demonstrated how _Fairlearn_ could be used to measure and mitigate
+# unfairness in the loan adjudication process.
+#
+# Using a dataset of credit loan outcomes (whether an individual defaulted on
+# a credit loan), we train a fairness-unaware model to predict the likelihood an
+# individual will default on a given loan. We use the *Fairlearn* toolkit for
+# assessing the fairness of our model, according to several metrics. Finally, we
+# perform two unfairness mitigation strategies on our model and compare the
+# results to our original model.
+#
+# Because the dataset used in the white paper is not publicly available, we
+# will introduce a semi-synthetic feature into an existing publicly available
+# dataset to replicate the outcome disparity found in the original dataset.
+
+# %%
+# Credit Decisions Dataset
+# -------------------------
+
+# %%
+# As mentioned, we will not be able to use the original loans dataset, and
+# instead will be working with a publicly available dataset of credit card
+# defaults in Taiwan collected in 2005. This dataset represents binary credit
+# card default outcomes for 30,000 applicants with information pertaining to an
+# applicant's payment history and bill statements over a six-month period from
+# April 2005 to September 2005, as well as demographic information, such as
+# *sex*, *age*, *marital status*, and *education level* of the applicant. A full
```

Comment:
The question that always comes to mind for me in such situations is "What about gender?" and "What made you choose sex instead of gender?"
There's an issue we filed #792 to document best practices when working with gender. If that existed yet I would have asked you to reference it, but alas... 
Perhaps a sentence saying that we didn't collect this data, but if we had we may have included other sensitive/demographic columns as well ? Wdyt?